src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -70,7 +77,11 @@ public EventTypeController(final EventTypeRepository eventTypeRepository,
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<?> createEventType(@Valid @RequestBody final EventType eventType,
                                              final Errors errors,
-                                             final NativeWebRequest nativeWebRequest) {
+                                             final NativeWebRequest nativeWebRequest)
+    {
+        if (featureToggleService.isFeatureEnabled("disable_event_type_creation")) {
```

Comment:
in parallel universe we have a long-living brach where we put these constants to FeatureToggleService class.
https://github.com/zalando/nakadi/blob/ARUHA-215-ut/src/main/java/de/zalando/aruha/nakadi/util/FeatureToggleService.java#L13